statsmodels/genmod/tests/test_glm_weights.py

```diff
@@ -251,8 +251,8 @@ def setup_class(cls):
         # in stata and here will make this less sketchy.
         cls.corr_fact = np.sqrt((wsum - 1.) / wsum) * 0.98518473599905609
         cls.res1 = GLM(cpunish_data.endog, cpunish_data.exog,
-                        family=sm.families.Poisson(), var_weights=aweights
-                        ).fit(cov_type='HC0') #, cov_kwds={'use_correction':False})
+                       family=sm.families.Poisson(), var_weights=aweights
+                       ).fit(cov_type='HC0') #, cov_kwds={'use_correction':False})
```

Comment:
this type I usually rewrite by defining 
mod = ...
res = mod.fit(...

Reply:
I like it!  Will update.